tests/foreman_api_test.py

```diff
@@ -0,0 +1,8 @@
+def test_foreman_organization(organization):
+    assert organization
+
+#def test_foreman_product(product):
+#    assert product
```

Comment:
for some reason this test fails, even tho the repository test below uses (internally) the same fixture and works fine :thinking: